### PR TITLE
Notification Indicator for Bookmarks

### PR DESCRIPTION
**Pull request by nmsturcke**
Add a notification indicator for bookmarks

Xm0TW

### CODE REVIEW

**Comment by nmsturcke** [comment]
Reverted extrapolated Notification Dot component, as per 
AlYCB

File: src/plugins/channelTabs/components/BookmarkContainer.tsx [hunk 1]
```diff
@@ -69,6 +69,24 @@ function Bookmark({ bookmarks, index, methods }: { bookmarks: Bookmarks, index:
     const bookmark = bookmarks[index];
 
     const ref = useRef<HTMLDivElement>(null);
+
+    let unreadCount, mentionCount;
+    if (!("bookmarks" in bookmark)) { // Not a folder
+        const channel = ChannelStore.getChannel(bookmark.channelId);
+
+        [unreadCount, mentionCount] = useStateFromStores(
```
Comment (sunnniee) [inline src/plugins/channelTabs/components/BookmarkContainer.tsx]: this conditionally calls a hook. you'd need to either move `useStateFromStores` out of the if statement and always return 0 for folders, or move this logic to `NotificationDot`

**Review by sunnniee — CHANGES_REQUESTED** [review]
oh fuck i forgot about this sorry

**Review by sunnniee — APPROVED** [review]
thank you!

**Pull request merged**